### ISSUE TITLE
Simplified UI's nametags setting broken

### ISSUE THREAD

**Issue by ctrlaltdavid** [vircadia/vircadia#182]
Simplified UI running.
Settings > General > Avatar Name Tags.
The options don't work.
[simplifiedUI.js] [UncaughtException signalHandlerException] Error: Result of expression 'nametag.handleAvatarNametagMode' [undefined] is not a function. in .../scripts/simplifiedUI/ui/simplifiedUI.js:117

**Comment by kasenvr** [comment]
Is this still an issue? I forgot if we fixed this.

**Comment by daleglass** [comment]
Seems like a very low priority since the scripting engine is due for a replacement, and simplified UI seems very little use, but leaving it here.

**Comment by Penguin-Guru** [comment]
I'm not sure what the "simplified UI" is but I noticed some noteworthy output while quitting the program. Is this related?

>[10/02 14:46:03] [DEBUG] [hifi.scriptengine] about to shutdown script: "nametags.js"
>[10/02 14:46:03] [DEBUG] [hifi.scriptengine] waiting on script: "nametags.js"
>[10/02 14:46:03] [DEBUG] [hifi.scriptengine] "nametags.js" stopAllTimers[ 0 ]
>[10/02 14:46:03] [WARNING] [hifi.scriptengine] [defaultScripts.js] Script.timerFired() while shutting down is ignored... parent script:defaultScripts.js
>[Previous message was repeated 92 times]
>[10/02 14:46:04] [WARNING] [hifi.scriptengine] Script Engine has been running too long, throwing: "nametags.js"
>[10/02 14:46:04] [DEBUG] [hifi.scriptengine] done waiting on script: "nametags.js"
>[10/02 14:46:04] [DEBUG] [hifi.scriptengine] about to shutdown script: "simplifiedNametag.js"
>[10/02 14:46:04] [DEBUG] [hifi.scriptengine] waiting on script: "simplifiedNametag.js"
>[10/02 14:46:04] [WARNING] [hifi.scriptengine] [defaultScripts.js] Script.timerFired() while shutting down is ignored... parent script:defaultScripts.js
>[Previous message was repeated 2 times]
>[10/02 14:46:04] [DEBUG] [hifi.controllers] Attempting to  disable  mapping  "hifi_simplifiedNametag"
>[10/02 14:46:04] [WARNING] [default] Setting::Manager::registerHandle(): Key registered more than once, overriding:  "com.highfidelity.experimental.enableExtendedJSExceptions"
>[10/02 14:46:04] [CRITICAL] [hifi.scriptengine] [simplifiedNametag.js] [UncaughtException signalHandlerException] Error: Can't find variable: blockedsKeyPressed in file:///tmp/.mount_VircadACUdn7/vircadia/interface/scripts/simplifiedUI/ui/simplifiedNametag/simplifiedNametag.js:94
>[10/02 14:46:04] [CRITICAL] [hifi.scriptengine] [Backtrace]
>[10/02 14:46:04] [CRITICAL] [hifi.scriptengine]     unload() at file:///tmp/.mount_VircadACUdn7/vircadia/interface/scripts/simplifiedUI/ui/simplifiedNametag/simplifiedNametag.js:94
>[10/02 14:46:04] [CRITICAL] [hifi.scriptengine]     <global>() at -1
>[10/02 14:46:04] [WARNING] [hifi.scriptengine] [defaultScripts.js] Script.timerFired() while shutting down is ignored... parent script:defaultScripts.js
>[Previous message was repeated 37 times]
